### DISCUSSION TITLE
List of "must haves"

### DISCUSSION THREAD

**Discussion by cosmo0** [benbarden/switchscores#49] [Ideas]
*Continued from https://github.com/benbarden/switchscores/issues/24#issuecomment-761549265*

I see people buying a Switch with like Mario or Zelda, and asking "what else should I buy?" and they want "AAA" games, not "interesting indies".

The "top 100 all time" is good, but it mixes "must haves" (Mario, Zelda...) with games that would more fit in a "indie darlings" or "hidden gems", like 9th Dawn III (it has a very high rank but only 3 reviews) and very niche games like Moero Crystal H (an erotic dungeon crawler).

I thought about a tag, but maybe it shouldn't be a tag, and rather be based on the score but also the number of reviews ? Like top 50 games that are also in the top 50 in number of reviews ? It would certainly make the "big games that everyone talks about" stand out.

@benbarden answered:

> "in the top 50 in number of reviews" is the part I'm unclear on.
> 
> To give you an idea, if I exclude any games with 6 or fewer reviews (I just chose 6 as the cutoff to try it out), these games are left out of the top 100: (I've not fully spaced it out)
> 
> ```
> id	title						game_rank	review_count
> 3449	Kentucky Route Zero: TV Edition			7		4
> 4641	9th Dawn III					12		3
> 2910	Divinity: Original Sin 2 - Definitive Edition	13		5
> 4703	Horace						17		6
> 623	Undertale					33		6
> 1420	Gone Home					33		3
> 3735	Children of Zodiarcs				33		3
> 4546	Moero Crystal H					33		4
> 5081	When the Past was Around			33		4
> 3557	Murder by Numbers				45		6
> 3109	Return of the Obra Dinn				50		5
> 3351	The Talos Principle: Deluxe Edition		50		5
> 5085	Dicey Dungeons					54		5
> 1478	TowerFall					59		3
> 4488	Mo:Astray					64		5
> 324	Deemo						73		4
> 1483	Into the Breach					73		6
> 3310	Assassin's Creed: The Rebel Collection		73		4
> 3938	Huntdown					73		6
> 665	The Banner Saga 3				87		5
> 1502	Forgotton Anne					87		5
> 8	Shovel Knight: Treasure Trove			93		6
> 434	Wandersong					93		6
> 3386	Nurse Love Addiction				93		3
> 3845	Yumeutsutsu Re:Master				93		3
> ```

I don't see many games that are really must-haves in this list.

Divinity 2 is very good but I rarely see anyone recommend it (and it's more a PC game). Assassin's Creed too, they're good games but getting old. Shovel Knight is a classic but not what I consider a "must have", it's still a retro indie.

The rest are indies, so it's not surprising they don't appear in a "must have" list. They're more "hidden gems", niche games, or "indie darlings".

What's the smallest number of reviews when you query the top 50 `order by review_count desc`?  
And then what do you get when you query the top 50 games `where review_count >= $reviews order by score desc` ? Just to see if the result has some sort of coherence.

**Comment by benbarden** [comment]
I'll look at the data when I get a chance, but I was thinking about this, and I think it's risky to remove games from the all-time list. I don't think that a game being hidden, niche, or indie should mean it doesn't qualify for the list.

But I do get your point, and I think my concern is partly down to this being difficult to put into words without it sounding like some games are worth more than others. I think the core of your idea comes from the number of reviews.

The all-time list is useful, and perhaps there could be additional lists as well:

- Top Rated - most popular (7 or more reviews)
- Top Rated - emerging favourites (3-6 reviews) or perhaps "Rising stars".

These are not necessarily the names to use, but you can see the split. Then, perhaps on the all-time list, we have a badge on each game that either says "Popular" or "Emerging" or something like that.

There is another list that might be useful here too - Most divisive. Basically, the games that have the widest spread of scores. I have a calculation for those numbers but it's buried in a unit test and I never actually added it to the site.

In short, let's not mess too much with the all-time list, but instead look at other lists that could be created to give more ways to see some of the top games on the Switch.

**Reply by cosmo0** [reply]
I wasn't clear: I didn't mean to remove or change the current "all-time bests"! I agree it's very useful ; just add a different list. "Top rated - most popular" sounds good! The "popular" badge is also a good idea.

Divisive is also a very good idea! It crossed my mind once or twice when I saw a few games with scores ranging from 3 to 9 but I never thought to write it down.

**Reply by benbarden** [reply]
OK, so we are saying the game thing :) Great. I can create the Most popular and Emerging favourites lists quite easily. The divisive list will take a bit longer to do but that can come later.
